3p/messaging.js

```diff
@@ -14,6 +14,52 @@
  * limitations under the License.
  */
 
+import './polyfills';
+import {dev} from '../src/log';
+
+/** @const */
+const AMP_MESSAGE_PREFIX = 'amp-';
+
+/** @enum {string} */
+export const MessageType = {
```

Comment:
I understand they are related this way. But non amp inabox host doesn't have to know all these MessageType.